pupil_src/shared_modules/gaze_mapping/gazer_2d_v1x.py

```diff
@@ -110,6 +152,7 @@ def predict(
     def filter_pupil_data(
         self, pupil_data: T.Iterable, confidence_threshold: T.Optional[float] = None
     ) -> T.Iterable:
-        # TODO: Filter 2D data
+        # TODO: Use topic to filter
+        pupil_data = list(filter(lambda p: "2d" in p["method"], pupil_data))
```

Comment:
Since the `filter` is converted to a `list` immediately, and the predicate is fairly short, a list comprehension could be more readable here (not a strong opinion).

Reply:
Fair point, will change it.